dataflux_pytorch/dataflux_iterable_dataset.py

```diff
@@ -114,15 +129,17 @@ def __iter__(self):
         worker_info = data.get_worker_info()
         if worker_info is None:
             # Single-process data loading.
-            yield from (self.data_format_fn(bytes_content) for bytes_content in
-                        dataflux_core.download.dataflux_download_lazy(
-                            project_name=self.project_name,
-                            bucket_name=self.bucket_name,
-                            objects=self.objects,
-                            storage_client=self.storage_client,
-                            dataflux_download_optimization_params=self.
-                            dataflux_download_optimization_params,
-                        ))
+            yield from (
+                self.data_format_fn(bytes_content)
+                for bytes_content in dataflux_core.download.dataflux_download_lazy(
+                    project_name=self.project_name,
+                    bucket_name=self.bucket_name,
+                    objects=self.objects,
+                    storage_client=self.storage_client,
+                    dataflux_download_optimization_params=self.dataflux_download_optimization_params,
+                    retry_config=self.config.download_retry_config,
+                )
```

Comment:
These reformatting might be coming from a different formatter that you used with the recommended one (checked in by Matt). Could you modify that? This way we can prevent modifying these lines consistently with different formatters.